linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml

```diff
@@ -9,6 +9,8 @@ description: |-
     Libreswan is supported by system crypto policy, but the Libreswan configuration may be
     set up to ignore it.
 
+    Either Libreswan is not installed, or it is configured to use the system-wide defined crypto policy.
```

Comment:
This doesn't go here. This goes into OCIL.

Reply:
Moved to OCIL.